dataflow/src/backlog/multiw.rs

```diff
@@ -26,6 +27,16 @@ impl Handle {
         }
     }
 
+    /// Evict `count` randomly selected keys from state and return them along with the number of
+    /// bytes freed.
+    pub fn empty_at_index(&mut self, index: usize) -> Option<&Vec<Vec<DataType>>> {
```

Comment:
I think this might see a stale version of the data at a given index. Remember how some writes aren't applied yet to the handle we receive after swapping?